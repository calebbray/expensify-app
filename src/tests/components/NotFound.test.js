import React from 'react';
import toJSON from 'enzyme-to-json';
import NotFound from '../../components/NotFound';
import { shallow } from 'enzyme';

test('should render not found page', () => {
  const wrapper = shallow(<NotFound />);
  expect(toJSON(wrapper)).toMatchSnapshot();
}) 