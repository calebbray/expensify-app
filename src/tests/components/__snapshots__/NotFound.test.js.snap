// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found page 1`] = `
<div>
  Error 404 Page Not Found 
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
